frontend/src/app/projects/models/model-detail/model-detail.component.html

```diff
@@ -42,4 +42,39 @@ <h1 class="title">Git Models</h1>
       </mat-card>
     </a>
   </div>
+  <h1 class="title">T4C Models</h1>
+  <div class="flexbox">
+    <a [routerLink]="['t4c-model', 'create']">
+      <mat-card matRipple class="mat-card-overview new">
+        <div class="content">
+          Use existing repository <br />
+          <div class="icon">
+            <app-mat-icon size="70px">add_circle_outline</app-mat-icon>
+          </div>
+        </div>
+      </mat-card>
+    </a>
+
+    <mat-card class="mat-card-overview new visually-disabled">
+      <div class="content">
+        Create repository <br />
+        <div class="icon">
+          <app-mat-icon size="70px">add_circle_outline</app-mat-icon>
+        </div>
+      </div>
+    </mat-card>
+    <a
+      *ngFor="let t4cModel of t4cModels"
+      [routerLink]="['t4c-model', t4cModel.id]"
+    >
+      <mat-card matRipple class="mat-card-overview">
+        <div class="header">Integration {{ t4cModel.id }}</div>
```

Comment:
For the git models I used the `gitModel.id`, because git models currently don't get assigned a name (which is not ideal). So we could use the actual name for the t4c models. However, it could also be confusing if the t4c models show their name while the git models just have `integration <id>`. What do you think about this?

Reply:
The name is not unique across all instances, and a model that has several T4C sources is likely to have two identically named sources on several instances for example. That’s why the name of the instance would be confusing there imo.